src/EditorFeatures/Core/BackgroundWorkIndicator/WpfBackgroundWorkIndicatorFactory.cs

```diff
@@ -20,11 +20,11 @@ namespace Microsoft.CodeAnalysis.Editor.BackgroundWorkIndicator;
 [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
 internal sealed partial class WpfBackgroundWorkIndicatorFactory(
     IThreadingContext threadingContext,
-    IToolTipPresenterFactory toolTipPresenterFactory,
+    IBackgroundWorkIndicatorService backgroundWorkIndicatorService,
     IAsynchronousOperationListenerProvider listenerProvider) : IBackgroundWorkIndicatorFactory
 {
     private readonly IThreadingContext _threadingContext = threadingContext;
-    private readonly IToolTipPresenterFactory _toolTipPresenterFactory = toolTipPresenterFactory;
+    private readonly IBackgroundWorkIndicatorService _backgroundWorkIndicatorService = backgroundWorkIndicatorService;
```

Comment:
Is _listener needed?

Reply:
never mind, I see it now

Reply:
actually, nope, I'm still confused. It looks like the ABWQ that was using it is gone now

Reply:
yes.  it's removed now.

Reply:
looks like ci found the ctor param can be removed too